examples/react-native/index.js

```diff
@@ -1,5 +1,11 @@
 import { AppRegistry } from 'react-native';
 import App from './App';
+import StorybookUIRoot from './storybook';
 import { name as appName } from './app.json';
 
-AppRegistry.registerComponent(appName, () => App);
+//TODO: replace with env var
+const isStorybook = true;
```

Comment:
Pulled down your branch and ran the following:
```
yarn clean && yarn && yarn build
yarn react-native-example ios:pod-install
yarn react-native-example ios
```

Leading to:
![Simulator Screen Shot - iPhone 13 - 2022-08-16 at 11 55 21](https://user-images.githubusercontent.com/15002885/184965575-7557750a-ea2a-435d-8806-6d7850edf4d2.png)

Think we may want to default `isStorybook` to `false` until we have wired up the env vars:

```suggestion
const isStorybook = false;
```

Reply:
Yeah sure, good catch. By the way regarding the error you are getting, we are using [storybook-loader](https://github.com/elderfo/react-native-storybook-loader#quick-start) to dynamically generate imports for all stories [as opposed to manually adding each story as we create them]. Running storybook will generate that file.

Reply:
Right, we should probably also build them in the scenario of running storybook without the web server

Reply:
Ack will add that in follow-up PR when wiring the env var